Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -357,6 +357,19 @@ def FlashRomImage(self):
         startup_nsh.WriteOut(nshpath, shutdown_after_run)
         VirtualDrive.AddFile(nshpath)
 
+        # Get the version number (repo release)
+        outstream = StringIO()
+        version = "Unknown"
+        ret = RunCmd('git', "rev-parse HEAD", outstream=outstream)
+        if ret == 0:
+            commithash = outstream.getvalue().strip()
+            outstream = StringIO()
+            ret = RunCmd("git", f'describe {commithash} --tags', outstream=outstream)

```

Comment:
If the repo is not directly on a tag, this will show the number of commits from the tag. It looks like you're directly using that value, right? If so, I think that's fine as it precisely describes the build version.

However, it might be helpful to breakdown that format in something like the readme in case people are unfamiliar with how to interpret the version value.

Reply:
git-describe is well documented in the official git docs. Is adding a note to go look at those docs fine?

Reply:
Yes, that's ideal. The goal is to direct someone who has never used `git describe` before in the right direction.

Reply:
Updated